src/desktop-lite/devcontainer-feature.json

```diff
@@ -47,6 +47,11 @@
             ],
             "default": "5901",
             "description": "Enter a port for the desktop VNC server (TigerVNC)"
```

Comment:
Can you bump minor version for this Feature? thanks! 

https://github.com/devcontainers/features/blob/08150ac2afba23ab494d66e31b499fa971e1a49e/src/desktop-lite/devcontainer-feature.json#L3

Reply:
Done in eff89e7 !